packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -25,6 +26,9 @@ export interface Props {
 
   /** If not set will call onDismiss if that is set. */
   onClickBackdrop?: () => void;
+
+  /** prevents users from tabbing away from items outside of the modal. You probably want this set to true, but making this opt-in for now as we test out this implementation */
+  shouldFocusTrap?: boolean;
```

Comment:
i am a bit reluctant in having this extra prop controlling the trap. if modals should always trap then i think it would make more sense to invest a little more into testing it now rather than introducing this extra prop. wdyt?

Reply:
@Elfo404 removed it! I had added it because I was originally using a different dependency for the focus traps and I think there was some concern that perhaps it wasn't stable enough to use everywhere. But now that I switched to FocusScope from react-aria I think it's no longer necessary. Removed!

Reply:
awesome!